shipit.stable.yml.sample

```
@@ -21,3 +21,4 @@ deploy:
     #    This will prevent publishes from being tagged as latest.
     # 2. Remove the following line, which updates Homebrew
```

Do we need to remove both of the following lines when this is no longer the latest stable version?

```suggestion
    # 2. Remove the following 2 lines, which update Homebrew
```